cardano-node/ChangeLog.md

```diff
@@ -25,13 +25,30 @@ None
   format will be supported for next two major releases of the node (the last
   major version which will support it is `1.37`). (#4563)
 
+- Fixed a tight loop when demoting a peer from hot to warm timeouts
+  (input-output-hk/ouroboros-network#4357).
+
 [#4563]: https://github.com/input-output-hk/cardano-node/issues/4563
 [understanding-config-files]: https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/understanding-config-files.md
 
+## 1.35.5 -- January 2023
```

Comment:
https://github.com/input-output-hk/cardano-node/releases/tag/1.35.5 
Can you ask somebody for the ledger team to update the change log and point to the PR that "addressed a fix in ledger internal data structures."